packages/limit-service/lib/limit.js

```diff
@@ -118,7 +118,51 @@ class FlagLimit extends Limit {
     }
 }
 
+class AllowlistLimit extends Limit {
```

Comment:
@tpatel I think this is a good direction to explore with a new type of Limit :+1:  

To know for sure how this interface should look like I'd look into how the configuration is generated for each Ghost instance. I'm suspecting current approach should work, but we should make sure adding a config like this one is possible and easy to do.